piksi_tools/console/settings.yaml

```diff
@@ -2199,3 +2199,202 @@
+  Notes:       The vector is measured in the device frame according to the markings on the device.
+
+- group: ins
+  name: antenna_offset_z
+  expert: false
+  type: double
+  units: meters
+  default value: '-0.12674'
+  readonly: false
+  Description: Z component of vector from device frame to antenna phase center  
+  Notes:       The vector is measured in the device frame according to the markings on the device.  The default
+               value represents the offset from the Duro Device Frame to the antenna phase center when the antenna
+               mounting bracket shipped with Duro is in use.
+
+- group: ins
+  name: vehicle_frame_offset_x
+  expert: false
+  type: double
+  units: meters
+  default value: '0'
+  readonly: false
+  Description: X component of vector from device frame to ins vehicle frame origin in which ins outputs are provided  
+  Notes:       The vector is measured in the device frame according to the markings on the device.  In order to output
+               ins solutions at the antenna phase center, this should match in sign and magnitude of antenna_offset_x
+               setting.
+
+- group: ins
+  name: vehicle_frame_offset_y
+  expert: false
+  type: double
+  units: meters
+  default value: '0'
+  readonly: false
+  Description: Y component of vector from device frame to ins vehicle frame origin in which ins outputs are provided  
+  Notes:       The vector is measured in the device frame according to the markings on the device.  In order to output
+               ins solutions at the antenna phase center, this should match in sign and magnitude of antenna_offset_y
+               setting.
+
+- group: ins
+  name: vehicle_frame_offset_z
+  expert: false
+  type: double
+  units: meters
+  default value: '-0.12674'
+  readonly: false
+  Description: Z component of vector from device frame to ins vehicle frame origin in which ins outputs are provided  
+  Notes:       The vector is measured in the device frame according to the markings on the device.  In order to output
+               ins solutions at the antenna phase center, this should match in sign and magnitude of antenna_offset_y
+               setting.  The default value represents vehicle outpus at the antenna phase center when the Duro antenna
+               mounting bracket is in use.
+
+- group: ins
+  name: constrain_vehicle_sideslip
+  expert: true
+  type: boolean
+  units: N/A
+  default value: false
+  readonly: false
+  Description: Enables or disables experimental non-holonomic constraint feature that allows INS to make assumptions about vehicle dynamics 
+  Notes:       This settings should only be enabled provided the vehicle frame Eueler angles are measured precisely and are correct.
+               It assumes that a vehicle can have no body velocity in the direction aligned with the vehicle "y" axis 
+               (i.e no sideslip).  This is a reasonable assumption for passenger vehicles and many tractors.
```

Comment:
suggested description:
```Enables or disables non-holonomic constraint that allows the inertial solution to make assumptions about vehicle dynamics```

suggested notes:
```This settings should only be enabled provided the vehicle frame Euler angles are measured precisely and are correct. It assumes a vehicle can have no velocity in the direction aligned with the vehicle "y" axis (i.e no sideslip). This is a reasonable assumption for passenger vehicles and many tractors.```

Should this be moved to an advanced setting since this is a non-supported feature?

Reply:
it is already advanced